Improve README for MySQL application (#571)
kanisterio/kanister: examples/stable/mysql/README.md

- Kubernetes 1.6+ with Beta APIs enabled
- PV provisioner support in the underlying infrastructure

> - Kanister controller version 0.26.0 installed in your cluster, let's assume in Namespace `<kanister-ns>`

Reviewer: ```suggestion
- Kanister controller version 0.26.0 installed in your cluster, let's assume in Namespace `kanister`
```

Reviewer: This will allow us to use the namespace in the examples below. Which makes it easy to copy paste

- Kanctl CLI installed (https://docs.kanister.io/tooling.html#install-the-tools)

## Installing the Chart

To install the MySQL database using the `stable` chart with the release name `mysql-release`:

```bash
# Add stable in your local chart repository
$ helm repo add stable https://kubernetes-charts.storage.googleapis.com/

# Update your local chart repository
$ helm repo update

# Install the MySQL database (Helm Version 2)
$ helm install stable/mysql -n mysql-release --namespace mysql-test \
    --set mysqlRootPassword='asd#45@mysqlEXAMPLE' \
    --set persistence.size=10Gi

# Install the MySQL database (Helm Version 3)
$ kubectl create namespace mysql-test
$ helm install mysql-release stable/mysql --namespace mysql-test \
    --set mysqlRootPassword='asd#45@mysqlEXAMPLE' \
    --set persistence.size=10Gi

```

> The command deploys an instance of MySQL in `mysql-test` namespace on Kubernetes cluster with default configurations.

Reviewer: ```suggestion
The command deploys a MySQL instance in the `mysql-test` namespace.
```

By default a random password will be generated for the root user. For setting your own password, use the `mysqlRootPassword` param as shown above.

> You can retrieve your root password by running the following command. Make sure to replace [MYSQL_RELEASE_NAME] and [MYSQL_RELEASE_NAMESPACE]:

Reviewer: Let's remove this and use the same namespace and release name as above ^^

Author: ```
You can retrieve your root password by running the following command. Make sure to replace [YOUR_RELEASE_NAME] and [YOUR_NAMESPACE]
```
we should go with this, right ?

>     `kubectl get secret [MYSQL_RELEASE_NAME] -n [MYSQL_RELEASE_NAMESPACE] -o jsonpath="{.data.mysql-root-password}" | base64 --decode`

Reviewer: Minor - let's use `--namespace` instead of `-n` - it's self documenting

s3-profile-drnw9   2m

# Create Actionset
# Replace kanister-ns with the namespace you have deployed the Kanister controller in
# Please make sure the value of profile and blueprint matches with the names of profile and blueprint that we have created already

> $ kanctl create actionset --action backup --namespace <kanister-ns> --blueprint mysql-blueprint --deployment mysql-test/mysql-release --profile mysql-test/s3-profile-drnw9 --secrets mysql=mysql-test/mysql-release

Reviewer: Let's use the namespace `kanister` here.
